css/style.css

```diff
@@ -89,6 +87,38 @@ video {
   background-color: #003c8f;
 }
 
+/* Small hero */
+.small-hero__image--quote {
+  background: url("../assets/images/IMG_8453.JPG") no-repeat center;
+}
+.small-hero__image--contact {
+  background: url("../assets/images/IMG_8056.JPG") no-repeat center;
+}
+.small-hero__image--gallery {
+  background: url("../assets/images/IMG_8056.JPG") no-repeat center;
+}
+
+.small-hero {
+  height: 200px;
+  background-size: cover;
+}
+
+.small-hero__layer {
+  height: 100%;
+  background-color: rgba(0, 60, 143, 0.5);
+}
+
+.small-hero__title {
+  font-family: "Roboto-Bold", sans-serif;
+  margin-top: -70px;
```

Comment:
The title needs to be 50px away from the bottom. I would suggest to not use negative margin to position the title. You could use instead absolute position and `bottom` to position it. If you choose to do this, remember that the parent element of this title would need `position: relative` so the title can be positioned correctly.

Reply:
Done